components/auth/routes.js

```diff
@@ -0,0 +1,66 @@
+const express = require('express');
+const passport = require('passport');
+const boom = require('@hapi/boom');
+const jwt = require('jsonwebtoken');
+const ApiKeysService = require('./apiKeysService');
+
+const { config } = require('../../config');
+
+// Basic strategy
+require('../../utils/auth/strategies/basic');
+
+function authApi(app) {
+  const router = express.Router();
+  app.use('/api/auth', router);
+
+  const apiKeysService = new ApiKeysService();
+
+  router.post('/sign-in', async function (req, res, next) {
+    const { apiKeyToken } = req.body;
+
+    if (!apiKeyToken) {
+      next(boom.unauthorized('apiKeyToken is required'));
+    }
+
+    passport.authenticate('basic', function (error, user) {
+      try {
+        if (error || !user) {
+          next(boom.unauthorized());
+        }
+
+        req.login(user, { session: false }, async function (error) {
+          if (error) {
+            next(error);
+          }
+
+          const apiKey = await apiKeysService.getApiKey({ token: apiKeyToken });
+
+          if (!apiKey) {
+            next(boom.unauthorized());
+          }
+
+          const { _id: id, firstName, email } = user;
+
+          const payload = {
+            sub: id,
+            firstName,
+            email,
+            scopes: apiKey.scopes,
+          };
+
+          const token = jwt.sign(payload, config.authJwtSecret, {
+            expiresIn: '15m',
+          });
+
+          return res
+            .status(200)
+            .json({ token, user: { id, firstName, email } });
```

Comment:
Multiline

Reply:
My prettier config is neccesary change this?